Apps/Src/Voltage.c

```diff
@@ -98,15 +99,30 @@ SafetyStatus Voltage_CheckStatus(void){
  * @return pointer to index of modules that are in danger
  */
 SafetyStatus *Voltage_GetModulesInDanger(void){
-	static SafetyStatus checks[NUM_BATTERY_MODULES];
-	uint32_t open_wires = Voltage_GetOpenWire();
+	static SafetyStatus checks[TOTAL_PINS];
```

Comment:
To better differentiate between boards and their pins, how does the following sound?
```suggestion
	static SafetyStatus checks[NUM_MINIONS][NUM_PINS_PER_IC];
```

Reply:
Yeah that would be a lot more clear